test/e2e_node/cpu_manager_test.go

```diff
@@ -115,8 +117,13 @@ func waitForContainerRemoval(ctnPartName string) {
 }
 
 func isHTEnabled() bool {
-	err := exec.Command("/bin/sh", "-c", "if [[ $(lscpu | grep \"Thread(s) per core:\" | cut -c24) != \"2\" ]]; then exit 1; fi").Run()
-	if err != nil {
+	outData, err := exec.Command("/bin/sh", "-c", "lscpu | grep \"Thread(s) per core:\" | cut -d \":\" -f 2").Output()
+	framework.ExpectNoError(err)
+
+	threadsPerCore, err := strconv.Atoi(strings.TrimSpace(string(outData)))
+	framework.ExpectNoError(err)
+
+	if threadsPerCore > 1 {
 		return false
```

Comment:
@flyingcougar `isHTEnabled()` should return `true` when `threadsPerCore > 1`. I would suggest that we remove this if block and the next `return true` statement and simply replace it with `return threadsPerCore >1`.